scripts/build.sh

```diff
@@ -50,6 +50,7 @@ if ! docker_tag="$(kudet get-docker-tag)"; then
     exit 1
 fi
 image_name="${IMAGE_ORG_AND_REPO}:${docker_tag}"
+image_name_no_version="${IMAGE_ORG_AND_REPO}:latest"
```

Comment:
nit: call this image_name_latest_version

Reply:
Good call, I took this name from other repos, but I guess `image_name_latest_version` is more explicit. Will change